pom.xml

```diff
@@ -98,7 +98,7 @@
   <properties>
     <arquillian.version>1.4.0.Final</arquillian.version>
     <junit.version>4.12</junit.version>
-    <spring-boot-bom.version>1.5.13.Final</spring-boot-bom.version>
+    <spring-boot-bom.version>1.5.13.SP1-SNAPSHOT</spring-boot-bom.version>
```

Comment:
I presume you have released a new BOM?

Reply:
Hmm, now that I look into it, I don't think we have a snapshot repository in the boosters so me releasing a snapshot of the BOM won't do any good here :)

Reply:
Yeah, tests seem to have failed catastrophically :P